src/licensedcode/data/rules/lbnl-bsd_13.RULE

```diff
@@ -0,0 +1,50 @@
+---
+license_expression: lbnl-bsd
+is_license_text: yes
+ignorable_copyrights:
+    - Copyright (c) 2003, The Regents of the University of California, through Lawrence Berkeley
+        National Laboratory
+ignorable_holders:
+    - The Regents of the University of California, through Lawrence Berkeley National Laboratory
+---
+
+Copyright (c) 2003, The Regents of the University of California, through
```

Comment:
Do not include the copyrights ... Start after "All rights reserved"

Reply:
Ack, removed.